kernel/declarations.mli

```diff
@@ -269,6 +269,13 @@ type mutual_inductive_body = {
 
     mind_nparams_rec : int;  (** Number of recursively uniform (i.e. ordinary) parameters *)
 
+    mind_params_rec_ctxt : Constr.rel_context;
+    (** The context of recursively uniform (i.e. ordinary) parameters.
+        Let-ins between the last uniform parameter and the first non-uniform
+        parameter are included in the uniform parameters. *)
```

Comment:
this seems highly redundant with mind_params_ctxt

Reply:
I'll remove it and replace it by call to split_uparams_nuparams